cmd/shell.go

```diff
@@ -109,7 +108,14 @@ func (c *ShellToContainerCmd) Run(context *Context, client *Client) error {
 	}
 	matches := httpHeaderRegexp.FindAllStringSubmatch(readStr, -1)
 	if len(matches) > 0 && len(matches[0]) > 1 {
-		return errors.New(strings.TrimSpace(readStr))
+		httpError, _ := strconv.Atoi(matches[0][1])
+		var message string
+		if (httpError == http.StatusNotFound) {
+			message = "App "+appName+" not found"
```

Comment:
Please use fmt.Sprintf instead of this concatenation.

Thanks for contributing!